docs/concepts/deposits-withdrawals.md

```diff
@@ -0,0 +1,116 @@
+
+Note: Associated and deposited are not equivalent, as deposited tokens are held within the ERC-20 bridge contract, and associated tokens stay in an Ethereum wallet or in the vesting contract.
+
+### Lifetime deposit limits 
+During alpha mainnet, the ERC-20 bridge smart contract limits how much can ever be deposited from an Ethereum address. This is done in an abundance of caution, to assure users in the face of recent bridge hacks that they would have only a small amount at risk at any point. 
+
+If, however, a user wanted to bypass those limits and understood the risks to their assets, they could run `exempt_depositor()` on the [ERC-20 bridge contract](./../api/bridge/contracts/ERC20.md) for each asset, after which transactions greater than deposit limit for the asset would be allowed.
+
+### Depositing ERC-20 assets
+Deposits go through the ERC0-20 bridge smart contract. Every type of asset supported by and voted into Vega will have a bridge, but for the time being there is only an ERC-20 bridge.
+
+When a participant wants to deposit assets onto a Vega key, they need to call a deposit function on the ERC20 bridge that contains how much of the specified asset the Vega key will receive.
+
+Once deposited, the assets are held in an asset pool for security and to make contract updates easier/less risky.
+
+:::note Go deeper
+Before running the deposit function, you must run the ERC20-standard approve function to authorise the bridge smart contract as a spender of the target token. This will only allow a specific amount of that token to be used by the bridge. This can be done directly or through a Vega app.
+
+Read about the ERC-20 token standard: [EIP-20: Token Standard proposal ↗](https://eips.ethereum.org/EIPS/eip-20)
+:::
+
+After a successful deposit transaction, wheter done directly or through Vega Console, the `Asset_Deposited` event will be emitted for use by the Vega event queue.
+
+The transaction is recognised by the Vega event queue and packaged as an event, which is then submitted to the validator nodes to verify the event contents against an Ethereum node that Vega validators also run.
+
+Once the transaction is verified, the Vega public key submitted in the transaction will be credited with the deposited asset.
+
+:::note Read more
+[ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#deposit_asset)
+:::
+
+### Diagram: Deposits
+![Deposit diagram](/img/concept-diagrams/diagram-deposit.png)
+
+## Withdrawing assets
+**Assets used for trading** and related activities can only be withdrawn if they're in a general account. That means that, among other things, they can't be in bond for liquidity provision or in a margin account for active orders. 
+
+**VEGA tokens** can only be withdrawn if they are not staked and/or locked.
+
+To remove assets from the Vega network, submit a withdrawal request via a Vega app, such as the Vega Console trading interface, or the API.
```

Comment:
similarly dso we need a separate unstaking link to another docs page or section here?